src/System.Collections.Concurrent/ref/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
       netcoreapp;
+      uap;
```

Comment:
With these we should also clean out the netcoreapp #ifdef's from the project and source files.